PubnativeLite/PubnativeLiteDemo/PNLiteDemoSettings.m

```diff
@@ -37,6 +37,7 @@
 NSString *const PNLiteDemoDFPBannerAdUnitID = @"/6499/example/banner";
 NSString *const PNLiteDemoDFPMRectAdUnitID = @"/6499/example/banner";
 NSString *const PNLiteDemoDFPInterstitialAdUnitID = @"/6499/example/interstitial";
+NSString *const PNLiteDemoAPIURL = @"https://api.pubnative.net";
```

Comment:
Why is the URL added twice? Isn't it already in the HyBid class? Probably we can read it from there. Or viceversa.

Reply:
i've added that as a constant to be consistent with all the params in that file & it needs a compile time initializer (static string in that manner)

Reply:
Ah cool then 😄